tools/taskcluster/get-sentry-token.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script talks to the taskcluster secrets service to obtain the
+Sentry token and write it to the .sentry_dsn_release file in the root
+directory.
+"""
+
+import os
+import taskcluster
+
+# Get JSON data from taskcluster secrets service
+secrets = taskcluster.Secrets({'baseUrl': 'http://taskcluster/secrets/v1'})
```

Comment:
Is there any reason not to combine the two scripts into one script? It'll make it easier to not duplicate code.

Reply:
I mainly did it this way because Focus does it this way, but is it possible that we wouldn't want both Sentry and Pocket in a build? These scripts could also be used for beta, but I guess you're right that we could do it together. I'll leave it as is for now.